_posts/2016-04-09-linux-mint-razer-blade.markdown

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: Running Linux Mint on a Razer Blade
+tags: linux mint razer-blade setup
+username: likethemammal
+---
+
+![Linux Mint logo](https://upload.wikimedia.org/wikipedia/commons/5/5c/Linux_Mint_Official_Logo.svg)
+
+**tl;dr** The <a  href="http://www.amazon.com/gp/product/B01AGINE7Q/ref=as_li_tl?ie=UTF8&camp=1789&creative=9325&creativeASIN=B01AGINE7Q&linkCode=as2&tag=likethemammal-20&linkId=VLVDTKUEZSESNTID">Razer Blade 2015</a><img src="http://ir-na.amazon-adsystem.com/e/ir?t=likethemammal-20&l=as2&o=1&a=B01AGINE7Q" width="1" height="1" border="0" alt="" style="border:none !important; margin:0px !important;" /> is a great portable gaming workstation. Linux Mint runs beautifully with some tweaking, that I've boiled down after months of usage. The only problem still unresolved is an issue with the internal webcam.
+
```

Comment:
I want to emphasize that it took me time and effort to get this list of tweaks together. It took months of trying commands, rebuilding from scratch, and frustration.

How would you suggest I break up the sentence and also get this across?